wger/weight/views.py

```diff
@@ -150,9 +150,29 @@ def overview(request, username=None):
                                      'month': max_date.month,
                                      'day': max_date.day}
 
+    last_five_weight_entries = WeightEntry.objects.filter(user=user).order_by('-date')[:5]
+    last_five_weight_entries_details = []
+
+    if last_five_weight_entries:
+        i = 0 
+
+        while (i != len(last_five_weight_entries)-1):
+            curr_entry = last_five_weight_entries[i]
+            prev_entry = last_five_weight_entries[i+1]
+            weight_diff = curr_entry.weight - prev_entry.weight 
+            if weight_diff > 0:
+                weight_diff = '+'+str(weight_diff)
```

Comment:
Actually, we don't need a helper function at all, in the view we can just do ``weight_diff = day_diff = None``. Then in the template we can check if the value is set, something like this

```
{% if entry_detail.1 %}
    {% if entry_detail.1 > 0 %}
        +
    {% endif %}
    {{ entry_detail.1 }}
{% else %}
    -/-
{% endif %}
```

This also makes is a bit clearer in the future to know where the "special" values + and -/- come from, since it is all done in the template